Code_of_Conduct.md

```diff
@@ -0,0 +1,65 @@
+
+Before Direct Messaging someone, ask for their permission on the group. If they allow you to, only then proceed to do so. The people in here are here to learn, explore and socialize in a professional environment.
+
+## Be inclusive
+
+
+All presentation material should be suitable for people aged 12 and above.
+
+Any public presentation which is part of any event, including but not limited to keynotes, presentations, lightning talks, addresses, mailing list posts, telegram posts and forums or any other form of communication that directly or indirectly indicates the involvement of the community, is subject to this code of conduct and thus may not contain:
+
+    - sexual or violent imagery;
+    - exclusionary language;
+    - harassment and sexist, racist, or exclusionary jokes;
+    - insults or ad-hominem attacks.
+
+Presenters are asked to avoid language which is not appropriate for an all-ages audience as much as possible. Please use English wherever possible.
+
+If the subject matter of the presentation cannot be presented adequately without including language that could be considered offensive, this should be pointed out in advance, at the beginning of the talk and in the schedule. Failing to do so, will lead to actions mentioned in ["What does that mean for me?"](#What-does-that-mean-for-me?) section.
+
+If presenters are unsure whether their material is suitable, they are encouraged to show it to the event’s speaker liaison or send the same for reviews to admins before their session.
+
+## Be aware.
+
+
+We ask everyone to be aware that we will not tolerate intimidation, harassment, or any abusive, discriminatory or derogatory behaviour by anyone at any event or online.
+
+Complaints can be made to the admins or teacher representative of the group meeting them in-person contacting them through direct messages. All complaints made to event admins will remain confidential and be taken seriously. The complaint will be treated appropriately with discretion. Should admins or teacher representative consider it appropriate, measures they may take include:
+
+    - the individual(s) may be told to apologise
+    - the individual(s) may be told to stop/modify their behaviour appropriately
+    - the individual(s) may be warned that enforcement action may be taken if the behaviour continues
+    - the individual(s) may be asked to immediately leave the venue and/or will be prohibited from continuing to attend the event
+    - the incident may be reported to the appropriate authorities
+    - the admins/organisers can ban the individual(s) for unforeseeable future
+
+## What does that mean for me?
+
+
+All participants, including event attendees and speakers must not engage in any intimidation, harassment, or abusive or discriminatory behaviour.
+
+Here are some examples of behaviours that have occurred at past events which are not appropriate:
```

Comment:
Spell check: `behaviors` instead of the one mentioned here.
I don't think ALiAS has faced any such thing in the past, right @thisisayush?
IMHO, it should be rephrased to `Here are some examples of behavior that are considered totally inappropriate:`.